constants/constants.go

```diff
@@ -12,6 +12,9 @@ const (
 	SectionUSPCO  SectionID = 10
 	SectionUSPUT  SectionID = 11
 	SectionUSPCT  SectionID = 12
+	SectionUSPTX  SectionID = 16
+	SectionUSPOR  SectionID = 15
```

Comment:
It is mentioned in the pull request title that Colorado is added but, files `sections/uspor/uspor.go` and `sections/uspor/uspor_test.go` as well as this constant's name, seem to hint that the two-letter state code refers to Oregon. Is the pull request title incorrect?